sound/soc/sof/ipc4-topology.c

```diff
@@ -1745,6 +1751,10 @@ static int sof_ipc4_route_free(struct snd_sof_dev *sdev, struct snd_sof_route *s
 	extension |= SOF_IPC4_MOD_EXT_DST_MOD_QUEUE_ID(dst_queue);
 	extension |= SOF_IPC4_MOD_EXT_SRC_MOD_QUEUE_ID(src_queue);
 
+	if (!strcmp(src_widget->widget->name, "kpb.14.1")) {
+		dev_err(sdev->dev, "in %s %d ylb, bind kpb to %s, extension: %#x\n", __func__, __LINE__, sink_widget->widget->name, extension);
```

Comment:
do you need to print the line?
Why extension is special and why it is relevant if the widget name is not `kpb.14.1'?

Reply:
Will remove it. This is a RFC, so I didn't clear the code too much. My fault.